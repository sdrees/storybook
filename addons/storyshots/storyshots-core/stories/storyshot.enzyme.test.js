import path from 'path';
import { mount } from 'enzyme';
import initStoryshots from '../dist/ts3.9';

initStoryshots({
  framework: 'react',
  configPath: path.join(__dirname, '..', '.storybook'),
  renderer: mount,
});
