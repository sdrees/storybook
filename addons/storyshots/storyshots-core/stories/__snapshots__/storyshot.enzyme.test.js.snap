// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Another Button with some emoji 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    <span
      aria-label="so cool"
      role="img"
    >
      😀 😎 👍 💯
    </span>
  </button>
</Button>
`;

exports[`Storyshots Another Button with text 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    Hello Button
  </button>
</Button>
`;

exports[`Storyshots Button With Text 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    Hello Button
  </button>
</Button>
`;

exports[`Storyshots Button with some emoji 1`] = `
<Button
  onClick={[Function]}
>
  <button
    onClick={[Function]}
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "border": "1px solid #eee",
        "borderRadius": 3,
        "cursor": "pointer",
        "fontSize": 15,
        "margin": 10,
        "padding": "3px 10px",
      }
    }
    type="button"
  >
    <span
      aria-label="so cool"
      role="img"
    >
      😀 😎 👍 💯
    </span>
  </button>
</Button>
`;

exports[`Storyshots Welcome MDX to Storybook 1`] = `
<Welcome
  showApp={[Function]}
>
  <button
    onClick={[Function]}
    type="button"
  >
    Welcome
  </button>
</Welcome>
`;

exports[`Storyshots Welcome to Storybook 1`] = `
<Welcome
  showApp={[Function]}
>
  <button
    onClick={[Function]}
    type="button"
  >
    Welcome
  </button>
</Welcome>
`;
